README.md

```diff
@@ -1,3 +1,42 @@
 # medic-builds-repo
 
 Design doc for builds repository.
+
+DDOCS are [pushed up by travis](https://github.com/medic/medic-webapp/blob/master/scripts/ci/push_to_staging.sh) with a specific name. These are then [pulled back down by api](https://github.com/medic/medic-api/blob/master/controllers/upgrade.js) and deployed by [horticulturalist](https://github.com/medic/horticulturalist).
+
+## Naming scheme
+
+Three valid examples: `medic:medic:3.0.0-beta.1`, `medic:medic:3.0.0`, `medic:medic:my-awesome-feature-branch`.
+
+The first two chunks between `:` is the namespace and the application name respectively. The namespace is just that: it has no logic relevant and is just used to scope things. The application name maps to the original ddoc name. It is what will be used to determine what to call the ddoc when deploying it.
```

Comment:
My main point here is just that the db and app name are both used (mapped from the build db doc name) when horti installs a build.  It's quite possible I'm misunderstanding something and it would be nice to review this stuff together since lots of moving parts here but I think my point is better illustrated on the other PR: https://github.com/medic/medic-api/pull/187/files#r134029805